rest/handler.go

```diff
@@ -654,6 +660,21 @@ func (h *handler) logRESTCount() {
 	h.db.DbStats.DatabaseStats.NumPublicRestRequests.Add(1)
 }
 
+// reportDbStats will report the public rest request specific stats back to the database
+func (h *handler) reportDbStats() {
+	if !h.shouldUpdateBytesTransferredStats() {
+		return
+	}
+	var bytesReadStat int64
+	h.response.reportStats(true)
+	// load the number of bytes read on the request
+	bytesReadStat = h.requestBody.GetBodyBytesCount()
+	// as this is int64 lets protect against a situation where a negative is returned from GetBodyBytesCount() function and thus decrementing the stat
+	if bytesReadStat >= 0 {
```

Comment:
Nit: Avoids `.Add(0)` call on the noop case, which is probably going to be common for GET requests and work we'd like to avoid doing.

```suggestion
	if bytesReadStat > 0 {
```